### DISCUSSION TITLE
Scalability & server sizing recommendations?

### DISCUSSION THREAD

**Discussion by liveman1** [openseadragon/openseadragon#2088] [Q&A] [answered]
Hi,

is there any experience on the amount of users and a recommended server sizing / architecture?
Is OSD intended for only a few users or could it serve for 1000s of users as well?
Would any image or caching server be required?
Or are there other stacks that should be used instead? 

thanks a lot!

**Comment by iangilman** [comment, marked as answer]
One of the great things about OpenSeadragon is that you can use it with an entirely static backend (if you do convert your images ahead of time). That means you can scale as large as you want, by using things like CDNs or load balancers.
